src/components/common/Tabs/TargetProfileTab/index.js

```diff
@@ -52,6 +66,7 @@ const TargetProfileTab = () => {
         </div>
       </div>
       <div className="separator" />
+      {!targets?.length ? <EmptyTargets /> : !matches?.length ? <EmptyConversations /> : <Chat />}
```

Comment:
hmm nested ternaries make me anxious but ok 😬

Reply:
Me too! Although I couldn't come up with something more straight forward. This is the other thing I came up with. WDYT?

`{!targets?.length  && <EmptyTargets /> }`
`{targets?.length && !matches?.length && <EmptyConversations />}`
`{targets?.length && matches?.length && <Chat />}`

Reply:
generally, I do something like this:

```
const getSomeContent = args => {
  if(condition_1) return <Component_1 />:
  if(condition_2) return <Component_2 />:
  ...
  return <Default_Component />
}

<>
  {getSomeContent(args)}
</>
```

...And if `getContent` gets too complicated I create a separate component.